### DISCUSSION TITLE
Disable popup until requested manually

### DISCUSSION THREAD

**Discussion by mfiano** [company-mode/company-mode#1182] [Q&A] [answered]
Hello, I'm trying to migrate from auto-complete here. I am wondering how I can prevent company-mode from popping up the completions as I type. Instead I want to manually invoke it with a global key. I never want to see the menu unless I request it. Is this possible like ac-auto-show-menu?

**Comment by dgutov** [comment, marked as answer]
https://github.com/company-mode/company-mode/issues/554.

And yes, you can bind `company-complete-common` to a key.
